encoders/src/main/scala/org/apache/spark/sql/execution/columnar/encoding/ColumnDeltaEncoder.scala

```diff
@@ -728,14 +723,16 @@ object DeltaWriter {
              |};
           """.stripMargin
         }
+        /* // Commented as [[Pradeep]], should be replaced by some other logger class.
         CodeGeneration.logDebug(
           s"DEBUG: Generated DeltaWriter for type $dataType, code=$expression")
+          */
```

Comment:
Has the org.apache.spark.Logging class also been moved?

Reply:
No, this class has not moved but we are having the dependency of the snappy-jdbc module so able to extends it here.